integration-tests/test/erc20.spec.ts

```diff
@@ -1,8 +1,11 @@
 import { Contract, ContractFactory, Wallet } from 'ethers'
 import { ethers } from 'hardhat'
-import { expect } from 'chai'
+import chai, { expect } from 'chai'
 import { GWEI } from './shared/utils'
 import { OptimismEnv } from './shared/env'
+import { solidity } from 'ethereum-waffle'
+
+chai.use(solidity)
```

Comment:
This should not be required, it gets automatically injected by the hardhat-waffle plugin AFAIK.

Reply:
Hmmm I was getting a local linting error so I added this. idk, gonna leave it